packages/block-library/src/audio/edit.js

```diff
@@ -102,35 +110,49 @@ class AudioEdit extends Component {
 		const { autoplay, caption, loop, preload, src } = this.props.attributes;
 		const { setAttributes, isSelected, className, noticeOperations, noticeUI } = this.props;
 		const { editing } = this.state;
-		const switchToEditing = () => {
-			this.setState( { editing: true } );
+		const toggleEditing = () => {
+			this.setState( { editing: ! this.state.editing } );
```

Comment:
It be nice to make the property `isEditing` and the function `toggleIsEditing` so that it's consistent with other blocks.

Reply:
This function also doesn't call `speak` like other blocks do.